app/styles/_uswds-theme.scss

```diff
@@ -9,7 +9,7 @@ in the form $setting: value,
 
 @use 'uswds-core' with (
   // Disable scary but mostly irrelevant warnings:
-  $theme-show-compile-warnings: false,
+  $theme-show-compile-warnings: true,
```

Comment:
```suggestion
```
You can drop this setting altogether. It's set to true by default. See https://designsystem.digital.gov/documentation/settings/#general-settings